### PR TITLE
chore: code and warnings cleanup

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: brd-ios/breadwallet/src/ApplicationController.swift [hunk 1]
```diff
@@ -61,9 +61,7 @@ class ApplicationController: Subscriber, Trackable {
     
     private var isReachable = true {
         didSet {
-            if oldValue == false && isReachable {
-                self.retryAfterIsReachable()
-            }
+            if oldValue == false && isReachable { self.retryAfterIsReachable() }
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ApplicationController.swift:64]: why

File: brd-ios/breadwallet/src/ApplicationController.swift [hunk 2]
```diff
@@ -75,8 +73,7 @@ class ApplicationController: Subscriber, Trackable {
         do {
             self.keyStore = try KeyStore.create()
             self.coreSystem = CoreSystem(keyStore: keyStore)
-        } catch let error { // only possible exception here should be if the keychain is inaccessible
-            print("error initializing key store: \(error)")
+        } catch { // only possible exception here should be if the keychain is inaccessible
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ApplicationController.swift:76]: please correct all the comments you see and make them start with a capital letter.

File: brd-ios/breadwallet/src/Constants/Functions.swift [hunk 3]
```diff
@@ -10,18 +10,20 @@ import UIKit
...
 /// Executes the callback on the given queue when the device becomes unlocked, or immediately if protected data is already available
 func guardProtected(queue: DispatchQueue, callback: @escaping () -> Void) {
-    if UIApplication.shared.isProtectedDataAvailable {
-        queue.async {
-            callback()
-        }
-    } else {
-        var observer: Any?
-        observer = NotificationCenter.default.addObserver(forName: UIApplication.protectedDataDidBecomeAvailableNotification, object: nil, queue: nil) { _ in
+    DispatchQueue.main.async {
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Constants/Functions.swift]: i would revert this completely for now

File: brd-ios/breadwallet/src/ApplicationController.swift (hunk 1 above)
Comment (dangelovska) [reply brd-ios/breadwallet/src/ApplicationController.swift:64]: For resolving Body Length Violation warning, there was one line more than it should be

Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/ApplicationController.swift:64]: this is unnecessary. can be reverted

**Review by kenan-mamedoff — APPROVED** [review]

**Pull request merged**